app/posts/smoking-cessation.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: collection
 title: Smoking cessation
```

Comment:
Random, and might not be a big deal. The post seems to use "smoking cessation" and "tobacco cessation" fairly interchangeably... does it need to be consistent?